benchcab/config.py

```diff
@@ -67,6 +67,28 @@ def validate_config(config: dict) -> bool:
     raise ConfigValidationException(v)
 
 
+def read_optional_data(config: dict):
```

Comment:
Add a docstring using numpy style. (autodocstring extension in VS Code can pre-populate for you, make sure to select numpy style on settings first).

In the description, try to explain why we aren't setting up the "name" optional input here.